GETTING_STARTED.md

```diff
@@ -0,0 +1,415 @@
+# Getting started
+
+In this document you will find a step-by-step guide on how to get Eunomia running in a local Minikube cluster.
+
+Eunomia is watching GitOpsConfig Custom Resources. They and their handling by Eunomia will be discussed in general.
+
+You will run two examples:
+- Eunomia acting upon local GitOpsConfig changes,
+- Eunomia watching for changes in a remote repository and acting upon them.
+
+## Prerequisites
+
+In order to build Eunomia, you'll need the following:
+
+- [Git](https://git-scm.com/downloads)
+- [Go 1.12+](https://golang.org/dl/)
+- [Docker](https://docs.docker.com/install/)
+- [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
+- Access to a Kubernetes cluster
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (for local development and testing)
+
+You will also have to [set up the GOPATH](https://github.com/golang/go/wiki/SettingGOPATH) env variable (the instructions are for Go 1.13 but it's the same for Go 1.12).
+
+### Installing on a Mac
```

Comment:
This seems to suggest to me like if eunomia could be only used on a Mac; should we also maybe add some info about Linux? if only to say "should work but not tested"? Though I had an impression that some people in our team do use Linux, maybe you could try asking around whether it's true and consulting with them if possible?

Reply:
I set Eunomia up on Linux some time ago, now I'm doing it again to make sure it still works. I'm also adding a general setup with links to vendor pages with instructions on how to install the prerequisites